src/app/components/cells/text/SelectableShortText.jsx

```diff
@@ -111,10 +111,6 @@ const SelectableShortText = props => {
     }
   };
 
-  const placeCaret = inputNode => {
-    const l = f.size(value);
-    maybe(inputNode).method("setSelectionRange", l, l);
-  };
```

Comment:
**[Question]** Ich denke diese Funktion war da, damit beim Wechsel in den Edit-Modus einer Shorttext-Zelle das Caret ans Ende gesetzt wird. Blöderweise macht er das nun nach jeder Eingabe da er ja immer wieder neu rendert. Oder liege ich hier falsch?

Ganz löschen entfernt halt auch das Feature, dass das Caret immer am Ende ist wenn ich anfange zu editieren. Es darf halt nur beim 1. render passieren.

Reply:
![grud_caret](https://user-images.githubusercontent.com/40287903/161201893-90213243-24da-4444-a07d-f9fdc41f2aff.gif)
Ja, ich denke auch dass das deswegen passiert, das löschen entfernt allerdings nicht das feature, da beim initialen render der "ShorttextEditCell" das caret automatisch im input an letzter stelle ist, egal wo man hinklickt. Das gilt auch für mehrmaliges editieren.

Reply:
Ah, ok. Dann war das ein Überbleibsel aus der alten Zeit :)
@hermann-p Kannst Du das kurz bestätigen zur Sicherheit? Nicht dass das doch noch für etwas gut war an das gerade keiner denkt.

Reply:
Thimo hat schon Recht. Früher war das nötig, weil Value- und Selectionstates innerhalb der Selectable-Cell waren, jetzt haben wir allerdings den State um eins hochgezogen, und dadurch die Funktionalität der Auto-Ausfüllfunktion stark vereinfacht. Dadurch wurde der `ref`-Hook jetzt bei jedem Value-Change wieder ausgelöst.
In der Edit-Zelle ist aber auch nochmal ein `placeCaret`, für den Fall dass die Autovervollständigung deaktiviert ist. Mit dem neuen Datenmodell feuert der jetzt aber auch korrekt, und einmal reicht aus.